en/flight_controller/intel_aero.md

```diff
@@ -28,9 +28,69 @@ OS_VERSION = Poky Aero (Intel Aero Linux Distro) 1.5.1-dev (pyro)"
 AIRMAP_VERSION = 1.8
 FPGA_VERSION = 0xc1
 ```
+## Setup Intel Aero using Ubuntu
 
+This chapter is based on the [Intel Aero wiki](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation). Follow the steps to install the OS and Intel Aero repository (The Yocto does not necessarily have to be upgraded first). The required material for those steps consists of:
 
-## Flashing
+1. Power supply (battery or network cable)
+1. Micro HDMI to HDMI cable to attach a monitor
+1. Micro USB3 to USB2 female adapter
+1. USB Hub to attach mouse and keyboard
+
+As soon as the steps under [Intel Aero repository](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#intel-aero-repository) are completed an aero kernel is installed. From there on make sure to always boot using this kernel.
+
+Follow the instructions to flash the BIOS, FPGA and Flight Controller. Edit the mavlink router config file to include the laptop IP as a UDP Endpoint. After all those steps are completed, the drone should automatically connect to the QGroundControl running on the laptop.
+
+In a next step ROS needs to be installed. Instructions on how to install ROS can be found [here](https://github.com/intel-aero/meta-intel-aero/wiki/05-Autonomous-drone-programming-with-ROS).
+
+### Realsense Camera
+
+1. Realsense SDK
+
+   Follow the steps to install the Realsense SDK listed on the [aero wiki](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#intel-realsense-sdk).
+   When cloning the repository, the legacy branch needs to be used for the R200 model. If the D435 or D415 is used, the master branch needs to be cloned. All other steps are the same and the branches can be just switched back and forth if the camera is changed.
+
+   If the Realsense R200 is used, it can already be started over a ROS node using:
+
+   ```
+   roslaunch realsense_camera r200_nodelet_default.launch
+   ```
+
+   If any D400 series camera is used, follow the next step to install a different ROS wrapper.
+
+1. ROS Wrapper for D400 series Realsense
+
+   Follow the instructions in the [Realsense/ROS documentation](https://github.com/intel-ros/realsense#step-3-install-intel-realsense-ros-from-sources) to install a catkin workspace and clone the Realsense software.
+
+   Install the udev rules using:
+   ```
+   sudo cp config/99-realsense-libusb.rules /etc/udev/rules.d/
+
+   sudo udevadm control --reload-rules && udevadm trigger
+   ```
+
+   Now the Realsense can be started over a ROS node using:
+
+   ```
+   roslaunch realsense2_camera rs_camera.launch
+   ```
+
+### Obstacle Avoidance
+
```

Comment:
"To run the PX4 obstacle avoidance software," - there is no other mention of obstacle avoidance software in this doc. Is this PX4 software or Intel add ons? I just wonder if we should have a brief sentence/link to info about aero obstacle avoidance. Otherwise this is good.